orchestra/project_api/views.py

```diff
@@ -176,13 +176,16 @@ def get_serializer(self, *args, **kwargs):
 
         return super().get_serializer(*args, **kwargs)
 
-    def get_queryset(self):
+    def get_queryset(self, ids=None):
         queryset = super().get_queryset()
+        if ids:
+            return queryset.filter(id__in=ids).order_by('-created_at')
```

Comment:
```suggestion
            return queryset = queryset.filter(id__in=ids)
```

Reply:
with this change, there will be only one exit/return point from the function, so following its control logic will be easier :)